frontend/src/components/MapPosition/MapPositionSection.tsx

```diff
@@ -0,0 +1,34 @@
+import { Typography } from '@equinor/eds-core-react'
+import { Mission } from 'models/Mission'
+import styled from 'styled-components'
+
+const MapPositionView = styled.div`
+    display: grid;
+    grid-column: 1/ -1;
+    gap: 1rem;
+`
+
+interface MissionProps {
+    mission?: Mission
+}
+
+export function MapPositionSection({ mission }: MissionProps) {
```

Comment:
Should we add the apiCaller and a new endpoint to get the position? That way each component calls for what it needs and decides how often it wants the information. I see that this also applies to the task table. So if we want this we should also add an issue for it

Reply:
That seems like a good approach. I have created issue #458 on this.